src/main/java/org/apache/flink/benchmark/RemoteChannelThroughputBenchmark.java

```diff
@@ -80,6 +84,7 @@ public void remoteRebalance(FlinkEnvironmentContext context) throws Exception {
         StreamExecutionEnvironment env = context.env;
         env.enableCheckpointing(CHECKPOINT_INTERVAL_MS);
         env.setParallelism(PARALLELISM);
+        env.getCheckpointConfig().enableUnalignedCheckpoints(!mode.equals("AlignedCheckpoint"));
```

Comment:
Good idea for re-using this benchmark :)

Before merging this, in order to preserve the benchmark continuity with the previous results, it would be a good idea to rename the `remoteRebalance` benchmark in the codespeed UI to `remoteRebalance.ALIGNED`.

Alternatively after merging, we can remove the `remoteRebalance` benchmark from the UI to clean it up.

Reply:
Either option sounds fine to me. But where is the codespeed UI and can I modify it?